test/integration/entity-spec.ts

```diff
@@ -43,12 +45,13 @@ describe("Entity", () => {
         const mockContext = new MockContext<string>({
             context: testData.input,
         });
-        await entity(mockContext);
+        const result = await entity(mockContext);
 
-        expect(mockContext.doneValue).to.not.equal(undefined);
+        expect(mockContext.done.callCount).to.equal(0);

```

Comment:
Maybe there shouldn't be a `done` property at all, right? Not sure if we _need_ `done` due to some inheritance constraint though.

Reply:
Yeah good point, I don't think we need it for the testing infrastructure anymore. I removed its usage, but it still needs to exist as a property in the mock context class until we upgrade the `@azure/functions` version for the same reason as above